package.json

```diff
@@ -157,7 +157,8 @@
     "applications": {
       "gecko": {
         "id": "FirefoxColor@mozilla.com",
-        "strict_min_version": "57.0a1"
+        "strict_min_version": "57.0a1",
+        "update_url": "https://testpilot.firefox.com/files/FirefoxColor@mozilla.com/updates.json"
```

Comment:
I think we might need to play with this URL on a per-build basis: The add-on is built specifically for each environment - i.e. prod, stage, dev, local - and so the update URL won't be the same for each